src/components/CommentsList.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../api";
+
+const CommentsList = () => {
+  const { article_id } = useParams();
```

Comment:
it would be useful to have a loading state here too as they sometimes dont come through for a while after the rest of the page is laoded